### PR TITLE
Refactor/gui v2

### PR DESCRIPTION
**Pull request by pranavxc**
## Change Summary

Provide summary of changes with issue number if any.

## Change type

- [ ] feat: (new feature for the user, not a new feature for build script)
- [ ] fix: (bug fix for the user, not a fix to a build script)
- [ ] docs: (changes to the documentation)
- [ ] style: (formatting, missing semi colons, etc; no production code change)
- [ ] refactor: (refactoring production code, eg. renaming a variable)
- [ ] test: (adding missing tests, refactoring tests; no production code change)
- [ ] chore: (updating grunt tasks etc; no production code change)

## Test/ Verification

Provide summary of changes.

## Additional information / screenshots (optional)

Anything for maintainers to be made aware of

### CODE REVIEW

**Review by wingkwong — COMMENTED** [review]
Are we really using `Api2.ts`?

**Comment by pranavxc** [comment]
> Are we really using `Api2.ts`?

Currently, we are not using it, I'd added it as a fetch version of SDK and planned to use it with the new GUI. Since interceptor we had to update I'd switched back to the Axios version for timebeing.

File: packages/nc-gui-v2/assets/img/temp/251.json [hunk 1]
```diff
@@ -0,0 +1 @@
+[{"city":"Kabul"}]
```
Comment (wingkwong) [inline packages/nc-gui-v2/assets/img/temp/251.json:1]: Dummy file?

**Review by wingkwong — COMMENTED** [review]
- lots of duplicate images. e.g. `oracle.jpeg`, `oracle.png`, `oracle.png.jpg`.
- Dummy images in`nc-gui-v2/assets/img/abcd/` ?

File: packages/nc-gui-v2/assets/img/temp/251.json (hunk 1 above)
Comment (pranavxc) [reply packages/nc-gui-v2/assets/img/temp/251.json:1]: have to clean up the assets and public folder

**Review by wingkwong — APPROVED** [review]
No further comments

**Pull request merged**